### PR TITLE
feat: add Brazilian Portuguese translations

### PR DESCRIPTION
**Pull request by ouuan**
@crcpucmg, thanks for your contribution!

It would be better if you can invite someone else to review these translations, or you can check them again. It would also be OK if you don't feel like checking again. Tell me when you think this is ready to be merged.

I have invited you to the organization. After accepting the invitation, you'll have write-access to this pull request. If you don't want to make modifications via git/github, you can send me the modified translation file.

### CODE REVIEW

**Comment by ouuan** [comment]
@all-contributors add @crcpucmg as a contributor for translations.

**Comment by crcpucmg** [comment]
I accepted the invitation. Thanks.
I will try to find a reviewer. In the meantime, I'll do another review,
just in case.


Em sáb., 12 de fev. de 2022 às 10:54, Yufan You ***@***.***>
escreveu:

**Comment by coder3101** [comment]
> Tell me when you think this is ready to be merged

Approving for fulfilling the merge criteria. Just in case crcpucmg cannot find a reviewer and we want to merge this PR.

**Review by coder3101 — APPROVED** [review]

File: translations/pt_BR.ts [hunk 1]
```diff
@@ -0,0 +1,3271 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="pt_BR" sourcelanguage="en">
+<context>
+    <name>AppWindow</name>
+    <message>
+        <source>Hot Exit</source>
+        <translation>Saída</translation>
```
Comment (swift-zym) [inline translations/pt_BR.ts]: @crcpucmg Does "Saída" mean "Hot Exit" in Portuguese?

https://www.collinsdictionary.com/dictionary/portuguese-english/saida

**Comment by crcpucmg** [comment]
About "Hot Exit" and "Saída": it was the best choice at the time.
The word "exit" can have two uses: as a verb (leave/go out), or as a noun
(output/way out).
I chose ("saída" as a noun) would be better for (backing up and) leaving
the editor.
It could receive an adverb like "abruptly" or "unexpectedly", but I left it
without.


Em qua., 16 de fev. de 2022 às 07:12, Yiming Zhang ***@***.***>
escreveu:

**Comment by coder3101** [comment]
@ouuan can we merge this?

**Pull request merged**